### PR TITLE
chore: update Screener triggers

### PR DESCRIPTION
**Pull request by layershifter**
This PR:
- corrects naming of branches/PRs
- enables running Screener for `master` branch to accept screenshots

### CODE REVIEW

File: build/screener/screener.config.js [hunk 1]
```diff
@@ -40,7 +40,10 @@ module.exports = {
     // failureExitCode: 0,
   }),
   ...(process.env.GITHUB_REF && {
-    branch: process.env.GITHUB_REF,
+    // GITHUB_REF can be:
+    // - refs/heads/feature-branch-1
+    // - refs/pull/2040/merge
+    branch: process.env.GITHUB_REF.split('/')[2],
```
Comment (miroslavstastny) [inline build/screener/screener.config.js:44]: What is the REF for master?

File: build/screener/screener.config.js [hunk 2]
```diff
@@ -40,7 +40,10 @@ module.exports = {
...
     commit: process.env.GITHUB_SHA,
```
Comment (miroslavstastny) [inline build/screener/screener.config.js:45]: Why is this inside `GITHUB_REF` check?

File: build/screener/screener.config.js (hunk 1 above)
Comment (layershifter) [reply build/screener/screener.config.js:44]: Should be `origin/heads/master`

File: build/screener/screener.config.js (hunk 2 above)
Comment (layershifter) [reply build/screener/screener.config.js:45]: You're right, it's redundant. Updated it

**Review by miroslavstastny — APPROVED** [review]

**Pull request merged**